test/gemm-microkernel-tester.cc

```diff
@@ -1345,16 +1345,20 @@ void GemmMicrokernelTester::Test(
 
   const size_t planes = 2;  // 4 bit is 2 planes - low nibbles and high nibbles
   const size_t k2 =  round_up_po2(k(), 2);  // tester assumes byte aligned rows
+
+  // TODO - Add bl to gen script.
+  const size_t bl = k2;
```

Comment:
this is a can of worms though. We need to discuss this in terms of what constraints are OK for this from API point of view. IMO we should pad `k` to make sure k2 % bl == 0. and some constraints what we already have on bl mainly min, max, and other rounding ones.